docs/developers/constants.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
This file already exists on the `master` branch, but the PR adds it anew, causing a conflict. 

However, the file on `master` needs updating indeed. Please update it instead.

Reply:
Ah I wasn't to sure, since the pr mentioned I should branch from the external branch. 
Should I merge the master in to resolve the issue or should I branch directly from the master?

Reply:
the `external` branch wasn't updated. I updated it a few hours ago so it now contains the `constants.md` file.

Reply:
Ah great makes things a lot easier, thank you 👍 
I just merged and resolved the conflicts :)